### PR TITLE
fix: RDCC-2102 IgnoreCase search for ccdcasetype and removed sequence

### PR DESCRIPTION
**Pull request by shreedhar-hmcts**
### JIRA link (if applicable) ###
RDCC-2102

### Change description ###
fix: RDCC-2102 IgnoreCase search for ccdcasetype and removed sequence

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/lrdapi/service/impl/LrdServiceImpl.java [hunk 1]
```diff
@@ -43,7 +43,8 @@
 
         } else if (StringUtils.isNotBlank(ccdCaseType)) {
 
-            serToCcdCaseType = serviceToCcdCaseTypeAssocRepositry.findByCcdCaseType(ccdCaseType.trim().toUpperCase());
+            serToCcdCaseType = serviceToCcdCaseTypeAssocRepositry
+                .findByCcdCaseTypeIgnoreCase(ccdCaseType.trim().toUpperCase());
```
Comment (SujeethKumarAri) [inline src/main/java/uk/gov/hmcts/reform/lrdapi/service/impl/LrdServiceImpl.java:47]: do we need to convert to Upper case as we are ignoring case ? Correct me if I am wrong :)

**Review by pksaxena01 — APPROVED** [review]
Approved

**Review by SujeethKumarAri — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/lrdapi/service/impl/LrdServiceImpl.java (hunk 1 above)
Comment (shreedhar-hmcts) [reply src/main/java/uk/gov/hmcts/reform/lrdapi/service/impl/LrdServiceImpl.java]: yes, removed upper case

Comment (SujeethKumarAri) [reply src/main/java/uk/gov/hmcts/reform/lrdapi/service/impl/LrdServiceImpl.java]: thanks!!

**Pull request merged**